app/presenters/flood_risk_engine/tabular_enrollment_detail_presenter.rb

```diff
@@ -27,10 +27,11 @@ def build_rows
         row_builder.organisation_type_row,
         row_builder.organisation_name_row,
         row_builder.organisation_address_row,
+        row_builder.registration_number_row,
         row_builder.correspondence_contact_name_row,
```

Comment:
`organisation_registration_number_row` feels more consistent and unambiguous here as the other org rows have that prefix